src/util/auth.ts

```
@@ -0,0 +1,5 @@
+import { Request, Response, NextFunction } from 'express'
+
+export const authMiddleware = (req: Request, res: Response, next: NextFunction) => {
+    return next()
```

TBD with revolt oauth

---

Note added to complete auth when OAuth2 comes out

Also made all the protected routes use it - if you want to disable the use of them until OAuth does come out, just return an error instead of next() (such as `return res.status(403).json({ error: 'Forbidden' }).end()`)

---

I gonna add oauth so ye dont worry